src/sentry/static/sentry/app/views/organizationEventsV2/data.jsx

```diff
@@ -67,11 +67,14 @@ export const SPECIAL_FIELDS = {
   event: {
     fields: ['title', 'id', 'project.name'],
     renderFunc: (data, org) => (
+      // TODO(mark) retain query string parameters so back
+      // ends up on search results also hacking the projectId in
+      // is janky
       <Link
         css={overflowEllipsis}
-        to={`/organizations/${org.slug}/projects/${data['project.name']}/events/${
+        to={`/organizations/${org.slug}/events/?eventSlug=${data['project.name']}:${
```

Comment:
We'll probably need to extend the existing location.query to keep the cursor, project, and other values around

Reply:
Oops, just saw your comment above